.circleci/config.yml

```diff
@@ -1231,9 +1231,20 @@ jobs:
       - update-spm-installation-commit
       - run:
           name: Submit Purchase Tester
-          working_directory: "Tests/TestingApps/PurchaseTesterSwiftUI"
```

Comment:
no longer needed or desired, we want the jobs to run from the root since that's where the framework is generated